mvpa2/base/node.py

```diff
@@ -281,7 +281,9 @@ def set_postproc(self, node):
     def __str__(self, *args, **kwargs):
         return _str(self, *args, **kwargs)
 
-    def __repr__(self, prefixes=[]):
+    def __repr__(self, prefixes=None):
+        if prefixes is None:
+            prefixes = []
```

Comment:
It was perfectly fine before. More concise code and no chance for --inplace changes since +[]. Why to fix not broken?